fcc-test-suite-template-master 2/index.html

```diff
@@ -8,25 +8,35 @@
 
   <body>
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
-    "Hello World, we are software developers!"
     <main id="main">
-      <h2 id="title">Dr. Norman Borlaug</h2>
-      <div id="img-div" style="display: flex; flex-direction: column; ">
```

Comment:
Kindly avoid using divs and utilize HTML5 semantics tags. Check this URL for more understanding:
<a href="https://www.w3schools.com/html/html5_semantic_elements.asp" target="_blank">Here</a>

Reply:
You did a great job adding a link to make the topic understandable but let's explain why we are requesting this change 👍